testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -818,6 +818,10 @@
   find("input[placeholder='Filter by Synopsis: ']").set(input)
 end
 
+When(/^I enter "([^"]*)" as the filtered channel name$/) do |input|
+  find("input[placeholder='Filter by Channel Name: ']").set(input)
```

Comment:
My bad thanks to port it :)

Reply:
No worry, you're not the only one to have forgotten ports.

That's the whole purpose of this operation.